pkg/cloudprovider/aws/cloudprovider.go

```diff
@@ -146,7 +146,7 @@ func (c *CloudProvider) GetInstanceTypes(ctx context.Context, provisioner *v1alp
 	if err != nil {
 		return nil, err
 	}
-	instanceTypes, err := c.instanceTypeProvider.Get(ctx, aws)
+	instanceTypes, err := c.instanceTypeProvider.Get(ctx, aws, provisioner)
```

Comment:
Thoughts on scoping this to v1alpha5.KubeletConfiguration?

Reply:
Sure, only thought was if we add details about VMOverhead, etc. later, we would potentially need the whole provisioner, but we can scope it down until we need it